accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -410,24 +420,29 @@ read.
     is treated as a first read and the initializer expression is evaluated
     again.
 
-A write to a field or variable which is marked `final` and `late` is a runtime
-error unless the field or variable was declared with no initializer expression,
-and there have been no previous writes to the field or variable (including via
-an initializing formal or an initializer list entry).
+Let `v` be a `late` and `final` non-local variable without an initializing
```

Comment:
This conflates the "variable", the name and the declaration. Which is probably common in the spec, but still confusing to me :)

Maybe change "non-local variable" to "non-local variable declaration", and change "`v`" to "*d*" (a metavariable denoting the declaration) and add "with name `v`" to link a name to the declaration. Use "*d*" below instead of "`v`" when referring to the thing that may or may not have been assigned.

Reply:
Done.